backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -57,6 +57,20 @@ def lambda_handler(event, context):
         "Resource": event["methodArn"],
     }
 
+    requested_resource = event["resource"]
+    path_params = event["pathParameters"]
+    request_method = event["httpMethod"]
+
+    logger.info(
+        f"Determining access for Resource: {requested_resource} "
+        f"- PathParams: {json.dumps(path_params) if path_params else 'N/A'} "
+        f"- Method: {request_method}"
+    )
+
+    if requested_resource.startswith("/app-config") and request_method == "GET":
```

Comment:
should this just be 
```suggestion
    if requested_resource == "/app-config" and request_method == "GET":
```

Reply:
I used `startswith` since for a GET request we expect the resource to look something like

```
/app-config?configScope&numVersions
```

but this does raise the question of whether I should just be explicit about only allowing 

```
/app-config?global&1
```

Since that's exactly what the request resource should be for getting the MLSpace-wide app config. I think this would be the correct way forward, I'm just going to do some testing.

Reply:
Good suggestions, I don't think the `startswith` are necessary here so I'll remove them in both places.